internal/driver/gomobile/driver.go

```diff
@@ -85,6 +88,7 @@ func (d *mobileDriver) CanvasForObject(fyne.CanvasObject) fyne.Canvas {
 	}
 
 	// TODO figure out how we handle multiple windows...
+	// TODO should use common.CanvasForObject as in desktop?
 	return d.currentWindow().Canvas()
```

Comment:
Any suggestion about this?

Reply:
I'm not sure what you mean. I agree with that the TODO states, it would fix an issue in mobile where the assumption about always being the top window is false.

Reply:
> I agree with that the TODO states, it would fix an issue in mobile where the assumption about always being the top window is false.

Got it.

Reply:
Unfortunately, I can't replace it with common.CanvasForObject as in desktop, because it causes a strange bug when you switch between two windows too fast (there is a possibility that the window is not correctly refreshed so you would see the previous window ui although you are in another window; resizing solves it, but the point is that it is not refreshed at time)